### PR TITLE
Enable LinearSvmBinaryClassifier

### LINKED ISSUE #160: Enable LinearSvmBinaryClassifier
Feature request from Azure AutoML team.

Add lines for LinearSvmBinaryClassifier to manifest_diff.json, similar to the following:
https://github.com/microsoft/NimbusML/blob/3b46629c6c1ed6e2b52847384d5474c1cc40648f/src/python/tools/manifest_diff.json#L230-L237

Run [entrypoint_compiler.py](https://github.com/microsoft/NimbusML/blob/master/src/python/tools/entrypoint_compiler.py) once with `--generate_entrypoints` flag and once with `--generate_api` flag.

Add examples using LinearSvmBinaryClassifier to [src\python\nimbusml\examples](https://github.com/microsoft/NimbusML/tree/master/src/python/nimbusml/examples) for both `FileDataStream` and `pandas.DataFrame`.

Add a test similar to the one for [AveragedPerceptronBinaryClassifier](https://github.com/microsoft/NimbusML/blob/master/src/python/nimbusml/tests/linear_model/test_averagedperceptronbinaryclassifier.py).

### PR DESCRIPTION
**Pull request by najeeb-kazmi**
Fix #160 

- Enable `LinearSvmBinaryClassifier`
- Add examples with `FileDataStream` and `pandas.DataFrame`
- Add test
- Update docs

### CODE REVIEW

File: src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+    @trace
+    def predict_proba(self, X, **params):
+        '''
+        Returns probabilities
+        '''
+        return self._predict_proba(X, **params)
+
+    @trace
+    def decision_function(self, X, **params):
```
Comment (ganik) [inline src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py]: >decision_function [](start = 8, length = 17)

Pls add tests for predict_proba() and decision_function()

Comment (najeeb-kazmi) [reply src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py]: Done

File: src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+from sklearn.base import ClassifierMixin
+
+from ..base_predictor import BasePredictor
+from ..internal.core.linear_model.linearsvmbinaryclassifier import \
+    LinearSvmBinaryClassifier as core
+from ..internal.utils.utils import trace
+
+
+class LinearSvmBinaryClassifier(
+        core,
```
Comment (ganik) [inline src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py]: Pls add this classifier to test_estimator_checks.py

Comment (najeeb-kazmi) [reply src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py]: @ganik I verified that LinearSvmBinaryClassifier checks are being executed in test_estimator_checks.py. There are no errors so no changes need to be made to test_estimator_checks.py.

Comment (ganik) [reply src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py]: Great!

---
In reply to: [302703328](https://github.com/microsoft/NimbusML/pull/180#discussion_r302703328) [](ancestors = 302703328)

**Pull request merged**